src/dbnode/storage/limits/query_limits_test.go

```diff
@@ -54,7 +56,11 @@ func TestQueryLimits(t *testing.T) {
 		Limit:    1,
 		Lookback: time.Second,
 	}
-	opts := testQueryLimitOptions(docOpts, bytesOpts, instrument.NewOptions())
+	seriesOpts := LookbackLimitOptions{
```

Comment:
should this be a separate test case?

Reply:
i decided to be consistent with the existing test, that was testing all the limits (docs + bytes)